frontend/scripts/pages/profile-place.page.js

```diff
@@ -164,7 +164,6 @@ const _setInfo = (info, nodeId) => {
   document.querySelector('.js-link-map').setAttribute('href', `./flows.html?selectedNodesIds=[${nodeId}]&isMapVisible=true&selectedYears=[${year},${year}]`);
   document.querySelector('.js-link-supply-chain').setAttribute('href', `./flows.html?selectedNodesIds=[${nodeId}]&isMapVisible=true&selectedYears=[${year},${year}]`);
   document.querySelector('.js-line-title').innerHTML = info.municipality ? `Deforestation trajectory of ${info.municipality}` : '-';
-  document.querySelector('.js-municipality').innerHTML = info.municipality ? info.municipality : '-';
```

Comment:
See https://github.com/Vizzuality/trase/pull/141/files#diff-d19ce98884d5ccf24a9d0388d66802a9R160

Reply:
It seems we're only using capitalize, we should import it from `lodash/capitalize`.